crates/swc_ecma_minifier/tests/terser/compress/evaluate/pow_nan/output.js

```diff
@@ -1 +1 @@
-console.log(NaN);
\ No newline at end of file
+console.log(Math.pow(1,NaN));
```

Comment:
It breaks test, is this intentional?
https://github.com/swc-project/swc/actions/runs/9904675638/job/27362648382?pr=9210